src/NUnitFramework/tests/Attributes/RangeAttributeTests.cs

```diff
@@ -52,7 +52,7 @@ public partial class RangeAttributeTests
         };
 
         // See XML docs for the ParamAttributeTypeConversions class.
-        private static readonly Type[] Int32RangeConvertibleToParameterTypes = { typeof(int), typeof(sbyte), typeof(byte), typeof(short), typeof(decimal) };
+        private static readonly Type[] Int32RangeConvertibleToParameterTypes = { typeof(int), typeof(sbyte), typeof(byte), typeof(short), typeof(decimal), typeof(long), typeof(double) };
```

Comment:
**NOTE: this change is required based on this line**: https://github.com/nunit/nunit/pull/3098/files#diff-64bcafa7925c912b792f366a7971facfR121

This was a hidden inconsistency between our attributes. `TestCaseAttribute` allowed `int -> long` and `int -> double` but `RangeAttribute` did not explicitly support the conversion. Now they are allowed on both (in nullable and non-nullable forms)

Reply:
This is good, we want this. The idea behind this list of special-cased conversions is to emulate C#'s implicit casts.